### ISSUE TITLE
error

### ISSUE THREAD

**Issue by MuskanMangla1** [milligram/milligram#373]
### Version info

**Milligram:**

**Other (e.g. normalize.css, node.js, npm, bower, browser, operating system) (if applicable):**

**Comment by uzair-ahmed-01** [comment]
yes i see its uses node.js 12 and its very important to mention all the version and campatiblity to easily setup the project adn work
